api/rs/slint/mcu.md

```diff
@@ -80,42 +76,42 @@ in a format that's suitable for the software based renderer we're going to use.
 
 ## Application Structure
 
-Typically, a graphical application in hosted environments is comprised of at least three different tasks:
+Typically, a graphical application in hosted environments has at least three different tasks:
 
- * Receives user input from operation system APIs.
- * Reacts to the input by performing application specific computations.
- * Renders an updated user interface and presents it on the screen using device-independent operating system APIs.
+ * Receive user input from operation system APIs.
+ * React to the input by performing application specific computations.
+ * Render an updated user interface and presents it on the screen using device-independent operating system APIs.
 
 The operating system provides an event loop to connect and schedule these tasks. Slint implements the
-task of receiving user input and forwarding it to the user interface layer, as well as rendering the user interface to the screen.
+task of receiving user input and forwarding it to the user interface layer, and rendering the user interface to the screen.
 
-In bare metal environments it is your responsibility to substitute and connect functionality that is otherwise provided by the operating system:
+In bare metal environments it's your responsibility to substitute and connect functionality that's otherwise provided by the operating system:
 
- * Select crates that allow you to initialize the chips that operate peripherals, such as a touch input or display controller.
-   If there are no crates, you may have to to develop your own drivers.
- * Drive the event loop yourself by querying peripherals for input, forwarding that input into computational modules of your
+ * Select crates that initialize the chips that operate peripherals, such as a touch input or display controller.
+   If there are no crates, you may have to develop your own drivers.
+ * Drive the event loop by querying peripherals for input, forwarding that input into computational modules of your
```

Comment:
Why did you remove the "yourself" part? I think that's quite essential in terms of conveying responsibility.